sequential/filter_fibonacci_numbers/src/filter_fibonacci_numbers.erl

```diff
@@ -3,4 +3,13 @@
 -export([filter/1]).
 
 filter(List) ->
-  put_your_solution_here.
+    FibList = fib_less_than(lists:max(List)),
+    [X || X <- List, lists:member(X, FibList)].
+
+fib_less_than(N) ->
+    Fig_gen = fun Fib(I, J, Acc) ->
+		      if I =< N -> Fib(J, I + J, [I | Acc]);
```

Comment:
if is kind of weird in erlang, try to avoid it using case instead. BUT a top level case like this is better expressed with different function clauses and pattern match.

Also I, J, N are not great argument names, can you use semething more expressive like Current, Next and such>

Reply:
* More stuff: `Fig_gen` is not proper Erlang naming.
* I'd prefer you add another helper with the extra N argument instead of using a lambda fun here.